Fix:

error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))'

error: implicit declaration of function 'read' is invalid in C99

error: implicit declaration of function 'write' is invalid in C99
--- cmain.c.orig	2005-10-04 07:28:47.000000000 -0500
+++ cmain.c	2020-11-16 12:20:44.000000000 -0600
@@ -15,6 +15,8 @@
  *  to me, then an attempt will be made to fix them.
  */
 
+#include <unistd.h>
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
--- configure.orig	2005-10-04 07:57:59.000000000 -0500
+++ configure	2020-11-16 12:18:14.000000000 -0600
@@ -2821,8 +2821,8 @@
   for (i = 0; i < 256; i++)
     if (XOR (islower (i), ISLOWER (i))
 	|| toupper (i) != TOUPPER (i))
-      exit(2);
-  exit (0);
+      return 2;
+  return 0;
 }
 _ACEOF
 rm -f conftest$ac_exeext
--- uncmain.c.orig	2005-10-04 07:28:47.000000000 -0500
+++ uncmain.c	2020-11-16 12:20:49.000000000 -0600
@@ -15,6 +15,8 @@
  *  to me, then an attempt will be made to fix them.
  */
 
+#include <unistd.h>
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
